system/vi/vi.c

```diff
@@ -858,10 +859,13 @@ static void vi_printf(FAR struct vi_s *vi, FAR const char *prefix,
 
   /* Expand the prefix message in the scratch buffer */
 
-  len = prefix ? snprintf(vi->scratch, SCRATCH_BUFSIZE, "%s", prefix) : 0;
+  if (prefix != NULL)
+    {
+      len = snprintf(vi->scratch, sizeof(vi->scratch), "%s", prefix);
```

Comment:
Do we need `Len = MIN(len, sizeof(vi->scratch)));` right after `snprintf` here?

Reply:
Done.